go.mod

```diff
@@ -1,13 +1,86 @@
 module gitlab.com/rackn/terraform-provider-drpv4
 
-go 1.13
+go 1.18
```

Comment:
`1.18` is out of support, we should update that

> Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).

https://go.dev/doc/devel/release

Reply:
I've updated to 1.20.